examples/tts/conf/fastpitch_ssl.yaml

```diff
@@ -0,0 +1,189 @@
+# This config contains the default values for training FastPitch model with aligner on LJSpeech dataset.
+# If you want to train model on other dataset, you can change config values according to your dataset.
+# Most dataset-specific arguments are in the head of the config file, see below.
+
+name: FastPitch
+
+train_dataset: ???
+validation_datasets: ???
+ssl_model_ckpt_path: ???
+hifi_ckpt_path: ???
+
+# LJSpeech stats (per frame)
+# ignored if pitch_normalization: speaker_wise
+pitch_mean: 212.35873413085938
+pitch_std: 68.52806091308594
```

Comment:
Could you replace these with `???` as defaults (with the LJSpeech stats as a comment)? We've decided to have the users set these themselves so they're aware they need to be changed depending on dataset used. (See #5392)

Reply:
done.